sources/engine/Stride.Rendering/Rendering/Images/MotionBlur/IMotionBlur.cs

```diff
@@ -0,0 +1,11 @@
+using Stride.Rendering.Images;
+
+namespace Stride.Rendering.Rendering.Images.MotionBlur
+{
+    public interface IMotionBlur : IImageEffect
+    {
+        bool NeedRangeDecompress { get; }
```

Comment:
Seems you're not referencing this property anywhere - also naming wise, should it follow the `Requires*` naming convention or is `Need` something else?

Reply:
I've copy pasted the fields from the anti aliasing interface but you're right this property is not used, i removed it in my last commit!